connectors/agentic-ai/src/test/java/io/camunda/connector/agenticai/adhoctoolsschema/processdefinition/feel/AdHocToolElementParameterExtractorTest.java

```diff
@@ -0,0 +1,82 @@
+package io.camunda.connector.agenticai.adhoctoolsschema.processdefinition.feel;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.assertj.core.api.Assertions.assertThatThrownBy;
+
+import io.camunda.connector.agenticai.adhoctoolsschema.model.AdHocToolElementParameter;
+import io.camunda.zeebe.feel.tagged.impl.TaggedParameterExtractor;
+import java.util.List;
+import java.util.Map;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.ValueSource;
+
+class AdHocToolElementParameterExtractorTest {
```

Comment:
This is now just a high-level test that the integration is working. The actual, detailed `fromAi()` test cases are now tested in [`zeebe-feel-tagged-parameters`](https://github.com/camunda/camunda/blob/main/zeebe/feel-tagged-parameters/src/test/java/io/camunda/zeebe/feel/tagged/impl/TaggedParameterExtractorTest.java).